### PR TITLE
add image plugin

### PR DESCRIPTION
**Pull request by pete911**

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

Thanks for submitting your kubectl plugin to Krew!
One of the krew-index maintainers will review it soon. Note that the reviews for new plugin submissions may take a few days.

In the meanwhile, here are a few tips to make your plugin manifest better:
* Make sure your plugin follows the [best practices](https://krew.sigs.k8s.io/docs/developer-guide/develop/best-practices/).
* Eliminate redundant wording form `shortDescription` (it should be max 50 characters).
* Try to word wrap your `description` to 80-character lines (no usage examples, please).

Thanks for your patience!
/kind new-plugin

**Comment by ahmetb** [comment]
This is almost what the "images" plugin does. Do you mind listing the differentiators?

**Comment by pete911** [comment]
Hi @ahmetb , thanks for quick reply. This plugin was build for specific needs. kubeclt-images output is very long when run agains large clusters (one line per pod). This plugin outputs one line per image. Summary of pods is displayed on the same line. This plugin shows also image size and id.

Some of the difference:
- pods per image (shorter output when running against large clusters) with additional information if any of these pods failed or restarted
  - this is useful to see immediately if there is image pull backoff (all pods would be restarted)
- image size - useful to see why startups are slow, if dockerfile needs to be reviewed etc.
- image id - easy to spot if images are being 're-tagged'. we would see the same tag but different id

File: plugins/image.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      sha256: 241593262a759c71c8d1dd8c0a4aa42277a60ab07846f515d2ebcbbc057a94a4
+      bin: kubectl-image.exe
+      files:
+        - from: kubectl-image.exe
+          to: .
+        - from: LICENSE
+          to: .
+  shortDescription: View images running in kubernetes cluster.
+  description: |
+    kubectl-image is a subcommand of kubectl to manage view images running in kubernetes cluster.
```
Comment (ahmetb) [inline plugins/image.yaml:83]: 1. No kubectl-foo syntax here pls plugins have "kubectl foo" syntax
2. Consider explaining functionality here. 
3. typo: "the Kubernetes cluster"
4. No need to link to docs, we print your homepage url in install/info commands anyway.

File: plugins/image.yaml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+          arch: arm64
+      uri: https://github.com/pete911/kubectl-image/releases/download/v1.0.10/kubectl-image_1.0.10_windows_arm64.zip
+      sha256: 241593262a759c71c8d1dd8c0a4aa42277a60ab07846f515d2ebcbbc057a94a4
+      bin: kubectl-image.exe
+      files:
+        - from: kubectl-image.exe
+          to: .
+        - from: LICENSE
+          to: .
+  shortDescription: View images running in kubernetes cluster.
```
Comment (ahmetb) [inline plugins/image.yaml:81]: - Kubernetes word is redundant
- Consider removing trailing "."

I'd probably rewrite this as

> Query container images by namespace/cluster.

**Review by ahmetb — COMMENTED** [review]
Cosmetic comments.

File: plugins/image.yaml (hunk 1 above)
Comment (pete911) [reply plugins/image.yaml]: thanks for detailed review, all points have been addressed

File: plugins/image.yaml (hunk 2 above)
Comment (pete911) [reply plugins/image.yaml]: updated short description

**Comment by ahmetb** [comment]
/lgtm
/approve
Thanks!
Also, please consider setting up [Krew release automation](https://krew.sigs.k8s.io/docs/developer-guide/release/automating-updates/) which helps you skip manually making updates to your Krew manifests on each new version and send a pull request. It’s a GitHub workflow bot that can run every time you push a new tag. Those PRs are automatically approved.

**Pull request merged**